sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/Models/KikChannelProperties.cs

```diff
@@ -31,13 +31,13 @@ public KikChannelProperties()
         /// Initializes a new instance of the KikChannelProperties class.
         /// </summary>
         /// <param name="userName">The Kik user name</param>
-        /// <param name="apiKey">Kik API key. Value only returned through POST
-        /// to the action Channel List API, otherwise empty.</param>
         /// <param name="isEnabled">Whether this channel is enabled for the
         /// bot</param>
+        /// <param name="apiKey">Kik API key. Value only returned through POST
+        /// to the action Channel List API, otherwise empty.</param>
         /// <param name="isValidated">Whether this channel is validated for the
         /// bot</param>
-        public KikChannelProperties(string userName, string apiKey, bool isEnabled, bool? isValidated = default(bool?))
+        public KikChannelProperties(string userName, bool isEnabled, string apiKey = default(string), bool? isValidated = default(bool?))
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?

Reply:
will add fallback constructor for back compat